### DISCUSSION TITLE
Missing Subscription Button

### DISCUSSION THREAD

**Discussion by karnos1000** [FreshRSS/FreshRSS#7119] [Q&A] [answered]
### Describe the bug

I don't have a Subscription Button. I am using FreshRSS 1.24.3 installed on TrueNAS ElectricEel-24.10.0.2. It's running as a docker container. I have tried this on multiple computers, in multiple browsers, and multiple versions of both TrueNAS and FreshRSS.  I had the button when I first installed the app, but after I added my first feed, it disappeared. I don't have it in Normal View or Reading View.

### To Reproduce

Please see attached image
![FreshRSS](https://github.com/user-attachments/assets/97f849b7-4082-43ee-9705-f3c202ad9595)

### FreshRSS version

1.24.3

### Environment information

- Database version: TrueNAS ElectricEel-24.10.0.2
- PHP version: Unknown
- Installation type: Docker
-Web server type: Unknown
- Device: Desktop using Vivaldi, Edge, Firefox, Samsung Galaxy Note Using Duck Duck Go
- OS: Windows 10, Latest Android
- Browser: Vivaldi 6.7.3329.31, Edge, Firefox,

**Comment by Alkarex** [comment, marked as answer]
Hello,
It looks like your are not logged in and are in anonymous mode.

**Reply by math-GH** [reply]
yes, it is the anonymous mode. See the link "About FreshRSS" instead of the subscription management button

**Comment by karnos1000** [comment]
Thanks for the help! I can't find any info in the documentation about
anonymous mode, or how to enter or exit it. Can you point me to where I
might find that information?
Thanks!

**Reply by math-GH** [reply]
anonymous mode means that the default user's feeds are public for not logged-in guest users.

More information see https://freshrss.github.io/FreshRSS/en/admins/13_Default_user.html
